src/configure/configure.ts

```diff
@@ -251,6 +251,10 @@ class PipelineConfigurer {
             }
         }
 
+        // Set working directory relative to repository root
+        let gitRootDir = await this.localGitRepoHelper.getGitRootDirectory();
+        this.inputs.pipelineParameters.workingDirectory = path.relative(gitRootDir, this.workspacePath);
```

Comment:
Should we add a check here whether workspacePath is subpath of gitRootDir? Otherwise we might have problem

Reply:
gitRootDir will either be same as workSpacePath or its parent. In what case workspacePath will not be a subpath of gitRootDir ?

Reply:
I don't know. But if there exists a case, user will see a wrong working directory. So, to stop that from happening we can show a proper error to user and log telemetry?

Reply:
I am okay with it if there is no change of this happening.

Reply:
Leaving this comment for this PR. Let's discuss tomorrow offline for this one.